cluster/gke/util.sh

```diff
@@ -155,12 +156,26 @@ function kube-up() {
     echo "... Using firewall-rule: ${FIREWALL_SSH}" >&2
   fi
 
-  local create_args=(
+  local pool_args=(
```

Comment:
Name is slightly misleading, since `num-nodes` and `machine-type` are also pool args. Maybe call this `shared_args` or something similar?